src/pages/explore/Explore.jsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import VideoCard from '../../components/card/VideoCard';
+import CategoryChips from '../../components/chips/CategoryChips';
```

Comment:
Can use barrel exports here too in the later stages

Reply:
sure ,will refactor it